src/Host.cpp

```diff
@@ -1147,6 +1147,38 @@ bool Host::installPackage( const QString& fileName, int module )
     }
     // reorder permanent and temporary triggers: perm first, temp second
     mTriggerUnit.reorderTriggersAfterPackageImport();
+
+    // This event was inspired by "Zoilder" in a Mudlet derivative in 2015
+    // Their version appended the packageName and "Event" to
+    // "sysInstallPackage", but I have revised it to pass the packageName as a
+    // separate argument.
+    // Due to the range of circumstances in which this method can be called
+    // I have also included:
+    // * the invoking "module" argument in case it is necesary to differentiate
+    // the way in which the installPackage code is called - I think this could
+    // be a factor in some use cases that Zoilder would not have experienced in
+    // their usage.
+    // * the original package filename
+    //
+    // The originators comment was (ES):
+    // Se ejecuta el evento que indica que se ha instalado el paquete.
+    // Así el propio paquete puede auto-inicializarse sin tener que reabrir la
+    // sesión.
+    // which I think translates to (EN):
+    // The event that indicates that the package is installed runs.
+    // Thus the package itself can auto-initialize without having to reopen the
+    // session.
+    TEvent installEvent;
+    installEvent.mArgumentList.append( tr( "sysInstallPackageEvent", "This string may not want/need to be translated" ) );
+    installEvent.mArgumentTypeList.append( ARGUMENT_TYPE_STRING );
+    installEvent.mArgumentList.append( packageName );
+    installEvent.mArgumentTypeList.append( ARGUMENT_TYPE_STRING );
+    installEvent.mArgumentList.append( QString::number( module ) );
```

Comment:
I understand where you're going with numbers - for the i18n focus - however between a number that's meaningless to _everybody_, even foreign users, and a word that is meaningful to _somebody_, that is our entire current userbase - and parts of the future userbase, because let's face it, if you're on the Internet, you'll be picking up a few written English words here and there - I think we should go with English words.

I don't think any human will be happy to deal with a nonsensical API that is all numbers at the end of the day versus words which at least look more different visually from one another.

I've already had to deal with this in the early versions of the exists API and it was _not fun_ to have an exit map translating non-sensical numbers to meaningful names all over the place. It was really unpleasant to work with.

Tagging some of our international users/experts @JorMox @Garagoth @keneanung  for opinions on this, happy to be swayed if others think it's the best.